en/admin-guide/data-export/index.rst

```diff
@@ -1,13 +1,204 @@
-Data export
-===========
+Export INTERLIS Data
+====================
 
-This represents a guide on how to export data from QGEP.
+This represents a guide on how to export data from QGEP to INTERLIS 2 Transfer Format (xtf).
 
-Export formats
-------------------------------
 
-INTERLIS :ref:`_export-interlis`
-* dxf (in progress)
-* Shape
-* MOUSE / MikeUrban (please connect with the developer team if you are interested in this)
-* other
+General
+^^^^^^^^^^^^^
+
+INTERLIS Export can be done using the ili2pg tool from Eisenhut Informatik and a series of prepared SQL scripts. They will do the following:
+
+* Translation Englisch – German / French (Classes / Attributes)
+* Conversion of value list to / from numeric values (Integer)
+* Restructuring of data from relational to object oriented modelling
+* Conversion of OID (16 characters) to foreign keys TID (integer)
+
+ .. figure:: images/interlisexport.jpg
+ 
+Step by step
+^^^^^^^^^^^^^
+
+Tool
+--------
+Download the latest version of `ili2pg <http://eisenhutinformatik.ch/interlis/ili2pg>`_
+
+ .. figure:: images/eisenhut.jpg
+
+Generate postgres ili2pg schema with data model
+-------------------------------------------------
+
+a) Generate schema with data model
+You need the wanted INTERLIS model with all related models:
+
+- *Units.ili*
+- *Base.ili*
+- *SIA405_Base.ili*
+- *VSA_DSS_2015_2_d_20170602.ili* for Version 2015 –> corrected version relations 
+
+>>> java -jar ili2pg.jar --createEnumTxtCol --schemaimport --importTid --sqlEnableNull --createEnumTabs --createFk  --noSmartMapping --dbdatabase qgep --dbschema vsa_dss_2015_2_d_391 --dbusr postgres --dbpwd sjib  --log createschema_VSA_DSS_2015_2_d_391.log VSA_DSS_2015_2_d_20170602.ili 
```

Comment:
Should be formatted as code (`.. code::` I think)

Reply:
No - I think this looks fine - see preview